controller/error.go

```diff
@@ -6,6 +6,22 @@ import (
 	"github.com/juju/errgo"
 )
 
+// ValidationError capsules validation errors into one error struct.
+// It is returned when the validation fails. causingErrors contains all
+// errors that occurenced, while validating the request.
+type ValidationError struct {
+	causingErrors []error
+}
+
+func (e ValidationError) Error() string {
+	// TODO better error message
+	msg := "group or unit invalid:\n"
```

Comment:
I'd like to find a better formatting here. An output might look like this currently:
```
inagoctl validate
Group 'invalid' not valid: group or unit invalid:
	group mixing scalable and non-scalable units
	unit does not have group prefix
```
I like to tabs, but redundant "group or unit invalid" is not nice. Anyone got an idea how to get a nicer message here?